services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -852,22 +854,47 @@ export class NoticeOfWorkApplication extends Component {
                 <LoadingWrapper condition={this.state.isTabLoaded}>
                   <div className={this.renderFixedHeaderClass()}>
                     <div className="inline-flex">
-                      <h2 className="padding-md">
-                        Referral
-                        <CoreTooltip
-                          title="This page allows you to identify and download the files that need to be included in the referral package.
+                      <h2 className="tab-title">
+                        <Popover
+                          placement="topLeft"
+                          content="This page allows you to identify and download the files that need to be included in the referral package.
             You may track progress on the E-Referrals website.
-            When responses are receives you can upload them by clicking on “Add Reviewer”
-            Finish this stage by clicking on “Complete Process” when all responses have been received.
-            If you need to make changes later, click “Resume Referral process”"
-                        />
+            When responses are receives you can upload them by clicking on “Add Referral”
+            Finish this stage by clicking on “Complete Referral Process” when all responses have been received.
+            If you need to make changes later, click “Resume Referral process”."
+                        >
+                          Referral
+                        </Popover>
                       </h2>
                       <NOWProgressActions tab="REF" />
+                      <ReferralConsultationPackage type="REF" />
                     </div>
                     <NOWProgressStatus tab="REF" />
                     <NOWStatusIndicator type="banner" tabSection="REF" />
                   </div>
-                  <div className="page__content">
+                  <div className={this.state.fixedTop ? "side-menu--fixed" : "side-menu"}>
+                    <NOWSideMenu
+                      route={routes.NOTICE_OF_WORK_APPLICATION}
+                      noticeOfWorkType={this.props.noticeOfWork.notice_of_work_type_code}
+                      tabSection="referral"
+                    />
+                    <a
+                      target="_blank"
+                      rel="noreferrer"
+                      href="https://j200.gov.bc.ca/int/ereferral/Default.aspx?PosseMenuName=EIMain"
```

Comment:
Can we store this as a constant somewhere? Perhaps in the `strings.js` file?

Reply:
moved it to a const!